tests.js

```diff
@@ -1,8 +1,9 @@
-require('./test/auth');
-require('./test/rabbitmq');
-require('./test/product');
-require('./test/dbBackup.test');
-require('./test/cleanup');
+require("./test/auth");
+require("./test/rabbitmq");
+require("./test/dbBackup.test");
+require("./test/product");
+require("./test/cleanup");
+require("./test/profile");
```

Comment:
The cleanup has to be the last test to run because it is cleaning the database :D

Reply:
I put it there on purpose because otherwise it introduced almost random bug :D but now I know..

FYI: If test fails weirdly, check the db to make sure it's completely dropped